src/components/Blinds/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import './style.css';
+
+import blindsclosed from './images/blinds-closed.svg';
+import blindsopen from './images/blinds-open.svg';
+
+const Blinds = ({ state }) => {
+  const [blinds, setBlinds] = useState(state);
+
+  return (
+    <div className="blinds">
+      <div className="blinds__icon">
+        {/*  <img src={blindsopen} alt="" /> */}
```

Comment:
vtěšinou zakomentovaný kód v gitu nechceme, protože máš k dispozici historii změn. Tj takovýhle zakomentovaný kód dělá pak jen bordel

Reply:
aha, oki, ja som si to neuvedomila ze tam vidim tu historiu zmien, a stale si to zakomentovavam. musim sa naucit lepsie pracovat s gitom v tomto smere